src/components/activity-header/header.tsx

```diff
@@ -36,7 +46,19 @@ export class Header extends React.PureComponent <IProps> {
     }
   }
 
-  private handleConcordLogoClick = () => {
-    window.open("https://concord.org/");
+  private renderActivityMenu = () => {
+    const { activityName } = this.props;
+    return (
+      <React.Fragment>
+        <div className={`activityTitle`} data-cy ="activity-title">Activity:</div>
+        <CustomSelect
+          items={[activityName]}
+          onSelectItem={(item: any) => console.log(item)}
```

Comment:
I would just pass an empty function or let onSelectItem be undefined.  We probably don't want a console.log here.  I just used that in my example so we could ensure it was working.